### ISSUE TITLE
Custom error message on required field not rendering when using themeConfig + resetting all classes

### ISSUE THREAD

**Issue by angelokeirsebilck** [verbb/formie#1040]
### Describe the bug

When setting a custom error message on required fields the error message does not get rendered when using the new themeConfig together with resetClasses.

The error message does get populated to the data-fui-message attribute but when you validate the form the default 'Please fill out this field.'  is used.

### Steps to reproduce

1. Set a custom error message on a required field.
2. Use the ready-made tailwind themeConfig
3. Validate the form

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Form template: Default Formie template

### Craft CMS version

4.2.0.2

### Plugin version

2.0.5

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.6](https://github.com/verbb/formie/releases/tag/2.0.6)
